theme/static/css/base.css

```
@@ -203,6 +232,30 @@ a:visited
   padding: 20px 0 0 676px;
 }
 
+.download-button {
+    display: flex;
+    padding: 14px 24px;
+    gap: 16px;
+    font-size: 24px;
+    color: #fff !important;
+    align-items: center;
+    background: #FFA210;
+    border-radius: 6px;
+    text-transform: uppercase;
+    font-family: Arial, sans-serif;
+    font-weight: 400;
+    text-decoration: none;
+}
+.download-button svg {
+    display: flex;
+    height: 1.125em;
```

the button is non-centered, the extra 0.125em going to the bottom of the button.
I would suggest to shift it back to central.

with my poor CSS skills, maybe sth like this?

```suggestion
    height: 1.25em;
    margin-top: -0.125em;
```

---

I'll rebase and fix that thx !

---

(btw, no need for rebase)

---

I needed to check first, and it appears the icon itself was misaligned.. 

So i've redone it, CSS is smaller.... and svg perfectly aligned :)